ui/src/components/shapes/circle.component.tsx

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) 2025 Intel Corporation
```

Comment:
The files from this folder are duplicated from Camilo's PR. Let's wait on that PR and rebase this onto that without the annotation shapes.